authors/settings.py

```diff
@@ -81,11 +82,19 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.getenv('database'),
+        'PORT': os.getenv('port'),
+        'USER': os.getenv('user'),
+        'PASSWORD': os.getenv('password'),
+        'HOST': os.getenv('host'),
     }
 }
 
+# Heroku DB config
+
+DATABASES['default'] = dj_database_url.config(os.getenv('DATABASES_URL'))
```

Comment:
This should be os.getenv('DATABASE_URL')

Reply:
Tackled. Thank you.